app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html

```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Alcohol Wholesaler Registration Scheme (AWRS) cancellation") {
```

Comment:
Please add the salutation to all templates, like in this other template you have:
https://github.com/hmrc/hmrc-email-renderer/blob/master/app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationEmailRegChange.scala.html#L3